### PR TITLE
Add troubleshooting_mode_enabled session variable.

### PR DESCRIPTION
**Pull request by stbof**
DOC-5147
DOC-5148

### CODE REVIEW

**Review by maryliag — APPROVED** [review]
:lgtm: 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14759#-:-N938jTqDut11URvG5MO:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained

File: _includes/v21.2/misc/session-vars.html [hunk 1]
```diff
@@ -839,5 +839,16 @@
    <td>Yes</td>
   </tr>
 
+  <tr>
+   <td>
+    <code>troubleshooting_mode_enabled</code>
+   </td>
+   <td>When enabled, avoid performing additional work on queries, particularly when the cluster is experiencing issues, unavailability, or failure. This session variable is used to avoid collecting and emitting telemetry data.</td>
```
Comment (taroface) [inline _includes/v21.2/misc/session-vars.html:846]: I think this is technically more correct:

```suggestion
   <td>When enabled, avoid performing additional work on queries, such as collecting and emitting telemetry data. This session variable is particularly useful when the cluster is experiencing issues, unavailability, or failure.</td>
```

File: _includes/v22.1/misc/session-vars.md [hunk 2]
```diff
@@ -56,6 +56,7 @@
...
 | <a name="transaction-rows-written-log"></a> `transaction_rows_written_log` | The threshold for the number of rows written by a SQL transaction. If this value is exceeded, the event will be logged to `SQL_PERF` (or `SQL_INTERNAL_PERF` for internal transactions). | `0` | Yes | Yes |
 | <a name="transaction-status"></a> `transaction_status` | The state of the current transaction. See [Transactions](transactions.html) for more details. This session variable was called `transaction status` (with a space) in CockroachDB 1.x. It has been renamed for compatibility with PostgreSQL. | `NoTxn` | No | Yes |
+| <a name="troubleshooting_mode_enabled"></a> `troubleshooting_mode_enabled` | When enabled, avoid performing additional work on queries, particularly when the cluster is experiencing issues, unavailability, or failure. This session variable is used to avoid collecting and emitting telemetry data. | `off` | Yes | Yes |
```
Comment (taroface) [inline _includes/v22.1/misc/session-vars.md:59]: Same suggestion as above

**Review by taroface — CHANGES_REQUESTED** [review]
Suggesting an edit for correctness

File: _includes/v22.1/misc/session-vars.md (hunk 2 above)
Comment (stbof) [inline _includes/v22.1/misc/session-vars.md:59]:
```suggestion
| <a name="troubleshooting_mode_enabled"></a> `troubleshooting_mode_enabled` | When enabled, avoid performing additional work on queries, such as collecting and emitting telemetry data. This session variable is particularly useful when the cluster is experiencing issues, unavailability, or failure. | `off` | Yes | Yes |
```

**Review by taroface — APPROVED** [review]
LGTM

**Pull request merged**